v2/guestbook-v2-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: extensions/v1beta1
```

Comment:
Why the new filename?

Reply:
-1 we should use the apps deployment path